components/media/media_channel/mod.rs

```diff
@@ -104,7 +104,7 @@ impl GLPlayerChan {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, Serialize, JSTraceable, MallocSizeOf)]
```

Comment:
@jdm 
How do I derive `JSTraceable` from `GLPlayerPipeline`?
`GLPlayerPipeline` is used in `DroppableField` for `HTMLMediaElement`.